src/commands/admin/sudo.ts

```diff
@@ -8,9 +8,32 @@ export default class extends Command {
 	description = 'Allows you always to run other commands.';
 	extendedHelp = 'Sudo bypasses permission checks, disabled command checks and command location checks.';
 	usage = '<command> [args]';
+	enabled = false;
```

Comment:
We discussed this in the testing server but it might be worth just excluding this from the PR outright for now

Reply:
I'll wait to take it out/merge the PR until the other Admins comment on it, since they haven't said anything on here or in the server yet.

Reply:
Cool cool

Reply:
yeah it's kind of an unnecessary headache at this point